doc/api/next_api_changes/behavior/17371-IHI.rst

```diff
@@ -0,0 +1,7 @@
+ioff and ion can be used as context managers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``plt.ion()`` and ``plt.ioff()`` may now be used as context managers to create
```

Comment:
This should be cross-referencable now that you've made the change to fix the docs?
```suggestion
`plt.ion()` and `plt.ioff()` may now be used as context managers to create
```